src/Common/MetricHelper.cs

```diff
@@ -292,6 +292,30 @@ public void SetPSHost(PSHost host)
         {
         }
 
+        private static void PopulateAuthenticationInfoFromQos(IEnumerable<IAuthenticationInfo> infos, IDictionary<string, string> eventProperties)
```

Comment:
Again I recommend use "Telemetry" instead of "Info" in names. Also "information" isn't countable so never use "infos".